src/app/modules/group/components/joined-group-list/joined-group-list.component.scss

```diff
@@ -0,0 +1,125 @@
+@import "src/colors.scss";
+@import "src/geometry.scss";
+
+:host ::ng-deep .ui-table .ui-table-thead > tr > th {
+  // position: -webkit-sticky;
+  // position: sticky !important;
+  // top: 5.8333rem;
+  // overflow: visible;
+}
+
+@media screen and (max-width: 64em) {
+  :host ::ng-deep .ui-table .ui-table-thead > tr > th {
+    top: 8.3333rem;
+  }
+}
+
+:host {
+  position: relative;
+  display: block;
+
+  ::ng-deep .ui-table {
+    margin-bottom: 1rem;
+    font-family: 'titillium', sans-serif;
+
+
+    .ui-table-thead > tr > th {
+      padding: 0 1rem;
+      text-align: left;
+      text-transform: uppercase;
+      font-size: .9rem;
+      letter-spacing: .03em;
+      height: 3rem;
+
+      .header-container {
+        width: 100%;
+        height: 100%;
+        position: relative;
+        display: flex;
+        align-items: center;
+
+          p-sorticon {
+            position: absolute;
+            right: 0;
+            top: 1.5rem;
+            transform: translateY(-50%);
+          }
+
+      }
+
+      &:focus {
+        outline: transparent;
+      }
+
+    }
+
+    .ui-table-tbody > tr:nth-child(even) {
+      background-color: #fff;
+    }
+
+    .ui-table-tbody > tr > td {
+      border: none;
+      border-bottom: 0.1rem solid #f0f0f0;
+      color: $base-text-color;
+      height: 4rem;
+      max-height: 4rem;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      font-size: 1.2rem;
+      padding : 0 1rem;
+    }
+
+    .ui-table-tbody > tr:last-child > td {
+      border-bottom: none;
+    }
+
+    .ui-table-tbody > tr.ui-state-highlight > td {
+      color: #fff;
+    }
+
+  }
+}
+
+
+
+
+.show-all {
```

Comment:
nothing should be out of the `:host` I think in scss files of components.

Reply:
fixed, I removed this part, it was not used anyway